src/Components/HabitTracker/HabitItem.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+
+const HabitItem = ({ title, duration }) => {
+  // Creating an array that will handle all the <div> and then we will display only this "progress" array
+  const progress = [];
+  // Creating strings to hold the classes and the colors
+  const bglightGrey =
+    'bg-lightGrey w-4 h-4  mx-auto rounded-full text-lg text-white flex items-center';
+  const bgprimary =
+    'bg-primary w-4 h-4  mx-auto rounded-full text-lg text-white flex items-center';
+  const bgred =
+    'bg-red-500 w-4 h-4  mx-auto rounded-full text-lg text-white flex items-center';
+
+  for (let i = 1; i <= duration - 1; i++) {
+    //the first push is for the rounded circle with the number in it
+    progress.push(
+      <div className="flex-1" key={i}>
+        <div
+          id={i}
+          onClick={() => {
+            if (document.getElementById(i).className === bglightGrey) {
```

Comment:
why are you using js in react, is this the only way to achieve the desired result